src/System.Runtime.Serialization.Formatters/src/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -2,23 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Reflection;
+using System.Diagnostics;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Reflection;
 using System.Runtime.CompilerServices;
-using System.Text;
-using System.Globalization;
-using System.Diagnostics;
 using System.Runtime.Serialization.Formatters;
+using System.Text;
 
 namespace System.Runtime.Serialization
 {
     public static class FormatterServices
     {
         private static readonly ConcurrentDictionary<MemberHolder, MemberInfo[]> s_memberInfoTable = new ConcurrentDictionary<MemberHolder, MemberInfo[]>();
 
-        private static FieldInfo[] GetSerializableFields(Type type)
+        private static FieldInfo[] InternalGetSerializableMembers(Type type)
         {
+            Debug.Assert(type != null);
```

Comment:
Although, by that logic, we would put them in far more places than we currently do.